assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -8407,4 +8554,789 @@ tests:
+              - document:pub
+              - document:pub1
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+              - user:charlie
+          - request:
+              filters:
+                - user
+              object: document:2
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+          - request:
+              filters:
+                - user
+              object: document:3
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:charlie
+          - request:
+              filters:
+                - user
+              object: document:pub
+              relation: viewer
+            expectation:
+              - user:*
+          - request:
+              filters:
+                - user
+              object: document:pub1
+              relation: viewer
+            expectation:
+              - user:*
+              - user:anne
+              - user:bob
+  - name: ttu_orphan_parent
+    stages:
+      - model: |
+          model
+            schema 1.1
+          type user
+          type group1
+            relations
+              define member: [user, user:*]
+          type group2
+            relations
+              define member: [user, user:*]
+          type document
+            relations
+              define parent: [group1, group2]
+              define viewer: member from parent
+        tuples:
+          - user: user:anne
+            relation: member
+            object: group1:1
+          - user: user:bob
+            relation: member
+            object: group2:1
+          - user: group2:1
+            relation: parent
+            object: document:1
+          - user: group1:1
+            relation: parent
+            object: document:1
+        checkAssertions:
+          - tuple:
+              user: user:anne
+              relation: viewer
+              object: document:1
+            expectation: true
+          - tuple:
+              user: user:bob
+              relation: viewer
+              object: document:1
+            expectation: true
+        listObjectsAssertions:
+          - request:
+              user: user:anne
+              type: document
+              relation: viewer
+            expectation:
+              - document:1
+          - request:
+              user: user:bob
+              type: document
+              relation: viewer
+            expectation:
+              - document:1
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+      # notice we are removing group2
```

Comment:
**Suggestion: ** Relocate this to line 9307

Reply:
9307 is a different test (ttu_orphan_public_wildcard_parent)

Reply:
I think you meant 9037